app/locale/en.js

```diff
@@ -4442,7 +4442,8 @@ module.exports = {
       questions: "Have questions or want more information? We'd be happy to help.",
       reach_out_manager: 'Reach out to your Account Manager __name__ at ',
       stats_include: 'stats include __number__ other __name__',
-      include_other_product: 'include __product__',
+      include_other_product: 'load __product__ stats',
+      show_other_product: 'show __product__ stats',
```

Comment:
why do we have show and include as different? It would be same, if you are loading it, then show it 😅 

Also, string comes out as `show ozaria stats`, it should be `Show Ozaria Stats`

Reply:
`include` start loading the ozaria data.  i don't want to load that again if the user want to `toggle` oz data. 
yes it loading by default. just add an option to temp hide oz data